cmd/kwil-admin/cmds/setup/genesis.go

```diff
@@ -0,0 +1,219 @@
+package setup
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/kwilteam/kwil-db/cmd/common/display"
+	"github.com/kwilteam/kwil-db/common/chain"
+	"github.com/spf13/cobra"
+)
+
+var (
+	genesisLong = `` + "`" + `genesis` + "`" + `creates a new genesis.json file.
+
+This command allows users to create genesis files without having to write them manually.
+
+If the ` + "`" + `--migration` + "`" + ` flag is set, an incomplete genesis file is generated that can be used in a zero downtime migration.
+If generating a migration genesis file, validators and initial state cannot be set.
+
+Validators, balance allocations, and forks should have the format "name:key:power", "address:balance", and "name:height" respectively.`
+
+	genesisExample = `# Create a new genesis.json file in the current directory
+kwil-admin setup genesis
+
+# Create a new genesis.json file in a specific directory with a specific chain ID and a validator with 1 power
+kwil-admin setup genesis --output /path/to/directory --chain-id mychainid --validator 890fe7ae9cb1fa6177555d5651e1b8451b4a9c64021c876236c700bc2690ff1d:1
```

Comment:
This validator example looks inaccurate.  Format in the code is "name:pubkeyhex:power"